eido/conversion_plugins.py

```diff
@@ -1,52 +1,36 @@
 """ built-in PEP filters """
-import sys
+from typing import Dict
 
 
-def basic_pep_filter(p, **kwargs):
+def basic_pep_filter(p, **kwargs) -> Dict[str, str]:
     """
     Basic PEP filter, that does not convert the Project object.
 
     This filter can save the PEP representation to file, if kwargs include `path`.
 
     :param peppy.Project p: a Project to run filter on
     """
-    path = kwargs.get("path")
-    if path is not None:
-        with open(path, "w") as text_file:
-            text_file.write(p.__str__)
-    else:
-        print(p)
+    return {"pep_conversion.txt": str(p)}
```

Comment:
I think of these as keys, not as file names. I don't see why you would include an extension here, and in the return values of the other filters

Reply:
Being discussed in #39